### DISCUSSION TITLE
Suggestion : Better FaceSwap feature

### DISCUSSION THREAD

**Discussion by Kaneki010** [lllyasviel/Fooocus#3189] [Ideas]
the current FaceSwap feature in Fooocus is extremely inferior to midjourney's , a lot of the time it doesn't even come close to replicating the inputted face . a way better possible alternative imo is inswapper 128 (a smaller version of midjourney's faceswap model) , and the low res results can be improved with a face restoration model like Codeformer or gfpgan

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/discussions/474#discussioncomment-9823803

**Reply by Kaneki010** [reply]
thanks a lot , im really happy knowing that inswapper's integration is possible
